Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingFeatureProcessor.cpp

```diff
@@ -51,7 +50,16 @@ namespace AZ
 
             // initialize the ray tracing buffer pools
             m_bufferPools = aznew RHI::MultiDeviceRayTracingBufferPools;
-            m_bufferPools->Init(RHI::MultiDevice::AllDevices);
+            m_bufferPools->Init(deviceMask);
+
+            auto deviceCount = RHI::RHISystemInterface::Get()->GetDeviceCount();
+
+            for (auto deviceIndex{0}; deviceIndex < deviceCount; ++deviceIndex)
```

Comment:
Do we need this data on all devices, or only those with raytracing support?

Reply:
Fixed